src/components/buttons/RemoveFromPlaylistButton.jsx

```diff
@@ -0,0 +1,24 @@
+import { useToast, useVideoState } from "../../context";
+import { removeFromPlaylist } from "../../services";
+
+export const RemoveFromPlaylistButton = ({ video, playlistId }) => {
+    const {_id} = video;
+  const { videoStateDispatch } = useVideoState();
+  const { showToast } = useToast();
+  return (
+    <div
+      onClick={() =>
```

Comment:
fix: You can create a function handleCLick and do these operations inside the function